charts/helm-project-operator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: helm-project-operator
 description: Helm Project Operator
-version: 0.0.1
-appVersion: 0.0.1
+version: 0.0.2
+appVersion: 0.0.2
```

Comment:
same question as below, we don't need to update the appVersion since we aren't releasing a new version of the helm-project-operator binary; we only need to update the version right?

Reply:
I answered in the previous comment - https://github.com/rancher/helm-project-operator/pull/16#discussion_r905865254.